crates/typst-kit/src/package.rs

```diff
@@ -178,4 +178,28 @@ impl PackageStorage {
             PackageError::MalformedArchive(Some(eco_format!("{err}")))
         })
     }
+
+    /// Purge the package cache.
+    pub fn purge_cache(&self) -> PackageResult<()> {
+        if let Some(cache_dir) = &self.package_cache_path {
+            if cache_dir.exists() {
```

Comment:
I think the _granularity_ needs to be discussed. I mean users might want to clear a specific package cache, instead of the whole cache directory.

BTW, because `package_cache_path` can be set via CLI args or environments, this recursive removing could be dangerous. If a malware set the `TYPST_PACKAGE_CACHE_PATH` environment variable to your `$HOME`, and you don't know it, the `purge` command can **silently** destroy your home...

Reply:
Well I mean, that's always going to be an issue, no? Even if I manually delete just the files, it'll still be dangerous

Reply:
At least, I think, let users confirm the being-removed directory.

Reply:
Yes that's a good idea, do we already have something similar?

Reply:
What about *in addition* rely on the known structure of this directory. So remove the subdirectory `packages/preview/` etc to ensure no data is accidentially deleted (with some certainty)

Reply:
> do we already have something similar?

I don't know if we have such infrastructure.